code/src/OSFramework/Feature/IRows.ts

```diff
@@ -7,7 +7,7 @@ namespace OSFramework.Feature {
         /**
          * Add a CSS class to a specific row from the grid.
          */
-        addClass(rowNumber: number, className: string);
+        addClass(rowNumber: number, className: string, refresh?: boolean);
```

Comment:
Some extra documentation would be useful. Such as, what is the "refresh" parameter for?

Reply:
done